crates/loader/src/local/assets.rs

```diff
@@ -260,3 +290,35 @@ fn is_absolute_guest_path(path: impl AsRef<Path>) -> bool {
     // paths specifically using Unix logic rather than the system function.
     path.as_ref().to_string_lossy().starts_with('/')
 }
+
+/// Convert strings to glob patterns
+fn convert_strings_to_glob_patterns<T: AsRef<str>>(files: &[T]) -> Result<Vec<glob::Pattern>> {
+    files
+        .iter()
+        .map(|f| {
+            glob::Pattern::new(f.as_ref())
+                .with_context(|| format!("can't convert {} to glob pattern", f.as_ref()))
+        })
+        .collect::<Result<Vec<glob::Pattern>>>()
+}
+
+/// Remove files which match excluded patterns
+fn get_included_files(files: Vec<FileMount>, exclude_patterns: &[glob::Pattern]) -> Vec<FileMount> {
+    files
+        .iter()
+        .cloned()
```

Comment:
I am not sure, but since you are passing `files` by value rather than by reference, you can probably to `into_iter` and save the clone?